app/models/miq_server.rb

```diff
@@ -406,7 +406,7 @@ def logon_status_details
 
   def ui_hostname
     if MiqEnvironment::Command.is_podified?
-      ENV.fetch("APPLICATION_DOMAIN")
+      ENV.fetch("APPLICATION_DOMAIN", nil)
```

Comment:
Any reason to use `fetch` instead of `[]`?

Reply:
https://docs.rubocop.org/rubocop/1.28/cops_style.html#stylefetchenvvar

> [Line 143](https://github.com/agrare/manageiq/blob/2667830ce8cc18fa7d30ae64bd9faed0fa24e51f/lib/container_orchestrator/object_definition.rb#L143), Col 56 - [Style/FetchEnvVar](https://docs.rubocop.org/rubocop/1.28/cops_style.html#stylefetchenvvar) - Use ENV.fetch("APPLICATION_DOMAIN") or ENV.fetch("APPLICATION_DOMAIN", nil) instead of ENV["APPLICATION_DOMAIN"].

I didn't in that location for consistency with the other code in the immediate area but for these I wanted to stay consistent with the rubocop recommendation

Reply:
This cop is specific to `ENV` (as opposed to all `Hash`) objects.  The premise is that if you are fetching an ENV var, you are likely expecting it to exist, so it's better to use `fetch` to show that intention.  It's one of those ones I'm on the fence on, but I understand rubocop's intention here.